protocol/xmrmaker/instance.go

```diff
@@ -134,7 +134,7 @@ func (inst *Instance) createOngoingSwap(s *swap.Info) error {
 		return inst.completeSwap(s, skA)
 	}
 
-	offer, err := inst.offerManager.GetOfferFromDB(s.ID)
```

Comment:
This function was pulling from the cache first, contrary to its name. It didn't seem like we needed it, when we already have `GetOffer`. `GetOffer` returns an error if the offer isn't found, which should be the behavior that we want here.